RSSReader/View/WebViewController.m

```diff
@@ -0,0 +1,172 @@
+//
+//  WebViewController.m
+//  RSSReader
+//
+//  Created by Uladzislau Stankevich on 7.01.21.
+//
+
+#import "WebViewController.h"
+#import <WebKit/WebKit.h>
+
+@interface WebViewController () <UIScrollViewDelegate>
+
+@property (nonatomic, assign)WKWebView *webView;
+@property (nonatomic, retain)UIBarButtonItem *backBarButtonItem;
+@property (nonatomic, retain)UIBarButtonItem *forwardBarButtonItem;
+@property (nonatomic, retain)UIBarButtonItem *refreshBarButtonItem;
+@property (nonatomic, retain)UIBarButtonItem *stopBarButtonItem;
+@property (nonatomic, retain)UIBarButtonItem *safariBarButtonItem;
+@property (nonatomic, retain)UIBarButtonItem *spacerBarButtonItem;
+@property (nonatomic) CGFloat contentOffsetLastY;
+
+@end
+
+@implementation WebViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.view.backgroundColor = UIColor.whiteColor;
+    [self setupLayout];
+    [self setupToolBar];
+    [self openURL];
+}
+
+- (instancetype)initWithURL:(NSURL *)url {
+    self = [super init];
+    if (self) {
+        _url = [url retain];
+    }
+    return self;
+}
+
+- (void)setupLayout {
+    [self.view addSubview:self.webView];
```

Comment:
adding view to the super view is not a part of the layout and should be done separately.

Reply:
fixed